src/DataBus/AzureDataBusPersistence.cs

```diff
@@ -28,6 +29,19 @@ protected override void Setup(FeatureConfigurationContext context)
             context.Services.AddSingleton(blobContainerClientProvider ?? new ThrowIfNoBlobServiceClientProvider());
             context.Services.AddSingleton<IDataBus>(serviceProvider => new BlobStorageDataBus(serviceProvider.GetRequiredService<IProvideBlobServiceClient>(),
                 dataBusSettings, new AsyncTimer()));
+            
+            context.Settings.AddStartupDiagnosticsSection(
+                "NServiceBus.Databus.AzureBlobStorage",
+                new
+                {
+                    dataBusSettings.UserProvidedConnectionString,
```

Comment:
Do not log the connection string.
```suggestion
```

Reply:
I am not, this is a boolean flag that remembers if the ConnectionString methods was called on the settings. I'm only logging the fact that this was done or not.

Reply:
Ah, the name is confusing. `UserProvidedConnectionString` implies "user-provided connection string". A name such as `UserHasProvidedConnectionString` would be easier to understand. Please disregard my comment.